File: Client/Jones.cpp/src/Money.cpp
#include <Money.h>
#include <string>
//constructors
CMoney::CMoney()
{

}

/*CMoney::CMoney(const CMoney &money)
{
	cash = money.cash;
	//this = money;
}*/

//Setters
void CMoney::setCash(int cash)
{
	money["cash"] = cash;
	//this->cash = cash;
}

void CMoney::setLoan(int loan)
{
}

void CMoney::setRent(int rent)
{
}

void CMoney::setGoods(int goods)
{
	//this->goods = goods;
	money["goods"] = goods;
}

void CMoney::setSallery(int sallery)
{
	//this->sallery = sallery;
	money["sallery"] = sallery;
}

void CMoney::setCurrency(const char *currency)
{
	strcpy_s(this->currency, "il");
	//TODO: fix the string copy
	//strcpy_s(this->currency,currency);
	//strcpy_s(this->currency,currency);
}

//Getters
int CMoney::getCash()
{
	return money["cash"];
}

int CMoney::getLoan(){return money["loan"];}
int CMoney::getRent(){return -1;}
int CMoney::getGoods(){return -1;}
int CMoney::getSallery(){return -1;}
char* CMoney::getCurrency(){return currency;}

File: Client/Jones.NET/Data/Money.cs
class Money
{
//private	map <string,int> money;
	private int cash; //How much cash the avatar has
	/*int loan; // how much loan the avatar took
	int rent; // how much rent the avatar needs to pay
	int goods; // how much goods he owns (Cloth + furniture + cet.)*/
	private int sallery; //How much money does he own per...(month, week, day?)
	private char[] currency; //what is the currency of the money

    public int Cash
    {
        get { return cash; }
        set { cash = value; }
    }


    public int Sallery
    {
        get { return sallery; }
        set { }
    }
	//Constructors
public 	Money(){}
	//CMoney(const CMoney &);

	//Setters
/*public 	void setCash(int i){}
public 	void setLoan(int i){}
public 	void setRent(int i){}
public 	void setGoods(int i){}
public 	void setSallery(int i){}
public void setCurrency(string s) { }*/

	//Gettters
/*public 	int getCash(){}
public 	int getLoan(){}
public 	int getRent(){}
public 	int getGoods(){}
public 	int getSallery(){}
public 	char getCurrency(){}*/
}